libr/util/uleb128.c

```diff
@@ -102,11 +102,16 @@ R_API ut8 *r_uleb128_encode(const ut64 s, int *len) {
 	return otarget;
 }
 
-R_API const ut8 *r_leb128(const ut8 *data, st64 *v) {
+R_API const ut8 *r_leb128(const ut8 *data, int datalen, st64 *v) {
```

Comment:
Hi! Could you add a unit test in test/unit/test_uleb128.c for this?

Reply:
👍

Reply:
The choice of `69` is not just for fun: it was a value for which the previous implementation incorrectly produced `-59`